python/tk_framework_desktopserver/shotgun/api_v2.py

```diff
@@ -766,10 +772,11 @@ def _process_commands(self, commands, project, entities):
 
         for command in commands:
             if command["app_name"] is not None:
+                self.logger.debug("Keeping command %s -- it has an associated app." % command)
```

Comment:
Note: I have a ticket in next sprint to do a cleanup/polish pass on logging for all of this. I'll take care of then when I do the rest of it for that ticket.